### ISSUE TITLE
About Figure 3(d)

### ISSUE THREAD

**Issue by xizaoqu** [jannerm/diffuser#34]
Hi~I found that in cases like Figure 3(a), one policy only applies to one maze, and needs references for reconstruction. Do cases in Figure 3(d) need references, or can be generalized to multiple scenes? By the way, have scripts for Figure 3(d) been released in this repo?

**Comment by jannerm** [comment]
What do you mean by references for reconstruction?

**Comment by xizaoqu** [comment]
> What do you mean by references for reconstruction?

Sorry for not making it clear. I mean to generate trajectories for a specific maze, we first need some ground truth trajectories (reference) (Figure 3(a)). It seems plausible to generate trajectories with a reward map (Figure 3(d)) and without reference. Did Figure 3(d) do in this way?

**Comment by jannerm** [comment]
The training dataset for those figures consists of random-walk trajectories in an enclosed square.
